fighters/common/src/general_statuses/shield/misc.rs

```diff
@@ -161,9 +166,22 @@ pub unsafe fn sub_guard_cont(fighter: &mut L2CFighterCommon) -> L2CValue {
         }
 
         if !guard_hold {
-            // <HDR>
+            // <HDR>\
+            // If your left stick is near the rim and you haven't triggered a roll
+            let escape_fb_stick_x = WorkModule::get_param_float(fighter.module_accessor, hash40("common"), hash40("escape_fb_stick_x"));
+            if fighter.global_table[STICK_X].get_f32().abs() > escape_fb_stick_x
+            && cat2 & *FIGHTER_PAD_CMD_CAT2_FLAG_STICK_ESCAPE_F == 0
+            && cat2 & *FIGHTER_PAD_CMD_CAT2_FLAG_STICK_ESCAPE_B == 0
+            && !VarModule::is_flag(fighter.battle_object, vars::common::status::ENABLE_UCF) {
+                // Enable UCF shielddrop thresholds
+                // change spotdodge y req from -0.75 to -0.8
+                // change platdrop y req from -0.71 to -0.675
+                VarModule::on_flag(fighter.battle_object, vars::common::status::ENABLE_UCF);
```

Comment:
If you enable UCF, I assume on the next frame it'll use the UCF drop threshold?

Reply:
> If you enable UCF, I assume on the next frame it'll use the UCF drop threshold?

Technically should be available on the same frame, but don't think it's possible to input both the shield tilt and shielddrop on the same frame lol